src/System.Runtime.Extensions/src/System/IO/StreamReader.cs

```diff
@@ -39,7 +32,6 @@ public class StreamReader : TextReader
         private Decoder _decoder;
         private byte[] _byteBuffer;
         private char[] _charBuffer;
-        private byte[] _preamble;   // Encoding's preamble, which identifies this encoding.
```

Comment:
Does the fact that this was being cached mean it's used multiple times?  I've not reviewed all of the StreamReader code, but it looks like we might now be accessing Encoding.Preamble multiple times.  In theory, could some custom encoding have a preamble and this change could now cause multiple calls to GetPreamble (via Preamble) where previously there was only one?  It's probably such a remote case that it's not worth caring about, but just wanted to check.